src/components/ListBox/README.md

```diff
@@ -0,0 +1,188 @@
+---
+name: ListBox
+category: Lists and tables
+keywords:
+  - list
+  - listbox
+  - interactive list
+---
+
+# ListBox
+
+The `ListBox` component is a list component that implements part the [Aria 1.2 ListBox specs](https://www.w3.org/TR/wai-aria-practices-1.2/#Listbox). It presents a list of options and allows users to select one or more of them. Through composition, it is possible to customize the presentation of these lists by using sections, headers or custom elements.
```

Comment:
If you need more structure than the standard component offers, use composition to customize the presentation of these lists by using sections, headers, or custom elements.